js/ui/codeView.js

```diff
@@ -265,6 +270,24 @@ var WindowTrackingButton = new Lang.Class({
     set state(value) {
         this.child.flipped = value == STATE_TOOLBOX;
     },
+
+    _onHoverChanged: function() {
```

Comment:
nitpick: Use the new method syntax: `_onHoverChanged() {` (here and below.)

Reply:
I agree with @cfoch here... Let's merge it with the consistent syntax since this is an old file. We can change it to use the new syntax all at once in a single pass later on.